README.md

```diff
@@ -1,2 +1,41 @@
-# openfisca-italy
-Italian tax and benefit system for OpenFisca.
+# Openfisca-italy
+[![Build Status](https://travis-ci.org/openfisca/openfisca-italy.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-italy)
```

Comment:
Why did you choose travis-ci ? We usually use cirleci in the projects and I'm curious :)

Reply:
We used travis, because we saw in OpenFisca-france [README.MD](https://github.com/openfisca/openfisca-france/blob/master/README.md) (3rd row) that you used it do indicate the project build state, but if it's better we will use [Circle CI embeddable status badge system](https://circleci.com/docs/1.0/status-badges/).

Reply:
You are right, I forgot it was there. We usually use Circle CI, it might me nice if we can share the testing configuration. In the end, it's up to you :)

Reply:
For us there aren't problems in using Circle Ci :) , do you have some hint on the string that we have to use for showing the Circle Ci status badge? <br />The default template link for get circle CI badge of default branch is: 
https://circleci.com/gh/:owner/:repo.svg?style=shield&circle-token=:circle-token
We substitute this link with this:
https://circleci.com/gh/openfisca/openfisca-italy.svg?style=shield&circle-token=:circle-token
and it's working but we aren't sure that it's correct. We await for your ack and then we will commit the changed README.md.